### ISSUE TITLE
EME6414 Week 2 #3 Hived Mind, Collective Intelligence, and Sci-Fi. | Shiyao Wei

### ISSUE THREAD

**Issue by sw3550** [sw3550/sw3550.github.io#3]
https://sw3550.github.io/post/qfeBPq686/ 

 Let's start this blog with Isaac Asimov. Currently, I'm engrossed in reading the Galactic Empire series as my bedtime re...

**Comment by vdennen** [comment]
I think you’ll find a lot of overlap between Internet concepts and SciFi. 😊
Vanessa

**Comment by sw3550** [comment]
> @vdennen
> I think you’ll find a lot of overlap between Internet concepts and SciFi. 😊
> Vanessa
> 

Yes, recently I've had even more ideas that go beyond this point. It's fascinating to think about how news from distant parts of the world can have such a significant impact on us. It's all thanks to the development of media technology. At the same time, people now have the ability to express their opinions and be heard by a much larger audience compared to half a century ago. Life itself sometimes is like Sci-Fi.
